src/util/validate.py

```diff
@@ -15,6 +15,20 @@
 }
 
 
+def validate_assessment_id(assessment_id):
+    """Validate that the provided assessment_id is matching the valid assessment_id format. Example: RV1234.
+
+    Args:
+        assessment_id (string): Assessment identifier to validate.
+
+    Returns:
+        match: the result of a regular expression match.
+    """
+    match = re.match(r"^RV\d{4}", assessment_id)
+
+    return match
```

Comment:
Since this is now a one-liner, we might as well eliminate the `match` variable altogether:
```suggestion
    return re.match(r"^RV\d{4}", assessment_id)
```

Reply:
See c6d29c5 for revisions